pkg/landscaper/controllers/installations/controller.go

```diff
@@ -367,6 +368,23 @@ func (c *Controller) initPrerequisites(ctx context.Context, inst *lsv1alpha1.Ins
 		return nil, lserrors.NewWrappedError(err, currOp, "SetupRegistries", err.Error())
 	}
 
+	if verify.IsVerifyEnabled(inst, c.LsConfig) {
```

Comment:
If you check the call hierarchy of the method initPrerequisites, this method is called twice during the processing of an installation, in the methods handlePhaseInit and handlePhaseCompleting. Would it perhaps be a possibility to verify the component just in the first call, e.g. by calling the methods with some boolean parameter to determine if verification should be executed or not.

Reply:
I added the suggested change. I think the method initPrerequisites should be refactored instead of adding a flag though.

Reply:
Perhaps I'm totally confused but you call the method always with `runVerify==false` and therefore a verification is never executed.

Reply:
could you please check again, I call it with true the first time
https://github.com/gardener/landscaper/pull/993/files#diff-766e33e443b4adbcff54a27f95dca2e50e0be398f347541bae6b62896dfd716bR239

Reply:
You are totally right. I missed that the method was called twice in the first method, one time with true.